src/include/os.h

```diff
@@ -126,6 +126,7 @@ struct __wt_file_handle_win {
     HANDLE filehandle_secondary; /* Windows file handle
                                     for file size changes */
     bool direct_io;              /* O_DIRECT configured */
+    int desired_access;          /* Read-only or read/write */
```

Comment:
It's a small thing but I think it is appropriate to use the `DWORD` which is an unsigned 32 bit int on windows.

Reply:
Thanks, fixed in c206f45. (The original function-local version was an int, and it was lazy of me to reuse that instead of checking the docs.)